org.entangle_photo.Manager.json

```diff
@@ -0,0 +1,127 @@
+{
+    "app-id": "org.entangle_photo.Manager",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "entangle",
+    "rename-icon": "entangle",
+    "rename-desktop-file": "entangle.desktop",
+    "rename-appdata-file": "entangle.appdata.xml",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=x11",
+        "--socket=wayland"
```

Comment:
IIUC, we'll need to add  "--device=all" so that the flatpak can access the USB device for the camera on the host.

Reply:
Also "--filesystem=xdg-pictures" so that it can read + write images.

It is possible we'll need --share=network in order to communicate with IP based cameras, though I'm not 100% sure on that one, as I'm not clear on network connectivity you get by default.

Reply:
> IIUC, we'll need to add "--device=all" so that the flatpak can access the USB device for the camera on the host.

Yeah, [is not ideal](http://docs.flatpak.org/en/latest/sandbox-permissions.html#device-access) but I guess for cameras we'll have to. I couldn't find a list of all the options that this receives to avoid passing "all". For now whlie testing I'm passing it by command line like this: `flatpak run --device=all ... `.

Reply:
You can't limit devices access for now

Reply:
if you want to access the camera, you might want to have `libusb`. Look at Darktable flatpak, it works there.